compiler/one-cmds/ODEFAULT.cfg

```diff
@@ -0,0 +1,99 @@
+# These options copied from 'onelib/constant.py`
```

Comment:
/cc @mhs4670go 

Any issue to support like this? if not, I'll make a new issue.

Reply:
Seems okay.